docs/pages/server-access/guides/teleport-ssh.mdx

```diff
@@ -0,0 +1,241 @@
+
+   ```bash
+   Nodename         UUID                                 Address        Labels                                
+   ---------------- ------------------------------------ -------------- ------------------------------------- 
+   ip-172-31-30-155 8f1e2719-3161-400f-b664-3279cf046511                env=example,hostname=ip-172-31-30-155 
+   ip-172-31-17-124 e7719967-b6eb-4fc0-8e53-00fac6d896d4 127.0.0.1:3022 env=example,hostname=ip-172-31-17-124 
+   ```
+
+5. Using the command `tsh ssh root@ip-172-31-17-124` locally, `auth-admin` can SSH into the `auth` Node and do the following:
+
+   i. Connect to other Nodes in the cluster by repeating the steps above.  
+   ii. Traverse the Linux file system.  
+   iii. Execute desired commands.  
+
+   - The `tsh ssh` command allows one to do anything they would if they were to SSH into a server using a third-party tool. 
+   - All commands executed by `auth-admin` are recorded and can be replayed in the Teleport Web interface.
+
+## Configure Single Sign-On
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/githubsso.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/githubsso.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+Teleport Server Access supports Single Sign-On (SSO) for authenticating users when they log into Teleport Web UI and through `tsh`. 
+
+This augments the enhanced security features that Teleport already offers with the robust security that SAML, multi-factor authentication, and SSO additionally provide.
+
+Specific set up guides are available for the following SSO providers:
+
+- GitHub SSO is described [here](../getting-started.mdx#step-3a-set-up-github-sso)
+- Gitlab SSO [here](../../enterprise/sso/gitlab.mdx)
+- Okta SSO [here](../../enterprise/sso/ssh-okta.mdx)
+- [Open ID Connect](../../enterprise/sso/oidc.mdx)
+- And [many more](../../enterprise/sso/ssh-sso.mdx)
+
+## Audit user and SSH sessions
+
+Teleport Server Access has built-in [session auditing](../../features/enhanced-session-recording.mdx) that will record single and joint user interactions in both audit log and video formats.
+
+- When a user authenticates and performs commands through `tsh ssh`, their activities are recorded in replayable session videos that can be viewed in Teleport Web UI.
+- Session recording captures each keystroke and command-line output.
+- Multiuser session auditing is also provided out-of-the-box. You can record remote IT sessions and administrative tasks without additional configuration.
+
+<video autoplay loop muted playsinline>
+  <source src="https://goteleport.com/teleport/videos/server-access/replay.mp4" type="video/mp4"/>
+  <source src="https://goteleport.com/teleport/videos/server-access/replay.webm" type="video/webm"/>
+Your browser does not support the video tag.
+</video>
+
+### Defining Events
```

Comment:
This will make more sense in architecture overview section of server access, where you would explain how events work.